Dockerfile

```diff
@@ -7,6 +7,6 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/*
 
 EXPOSE 9090
-ENV db_hostname dcsa_db
+ENV DB_HOSTNAME /db
```

Comment:
`/db` is not a valid hostname.  I guess you want to abuse the "empty" host and pass "context" parameters to the postgres URL string.  I do not mind that but I think the name is wrong.

Reply:
Yes, checking it.

Reply:
so `/db` is a valid hostname (at least in this context) since we use docker-compose to build the images, it uses a default network bridge that uses the service names mentioned in the docker-compose file.